docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+# Scalar DL Backup Creation and Restoration
+
+Scalar DL backup should be created after the Scalar DL server is paused otherwise there is a chance for inconsistency. 
+This guide shows you how to create and restore Scalar DL data backup without any data inconsistency.
```

Comment:
```suggestion
This guide shows you how to create and restore Scalar DL backups without any data inconsistency.
```

Reply:
Thank you for your suggestion.
Suggestion integrated cd32752